app/helpers/ui_helper.rb

```diff
@@ -33,22 +33,31 @@ def label_with_icon(icon, label_tag, position)
     ordered.compact.join.html_safe # rubocop:disable Rails/OutputSafety
   end
 
-  def input_field(form: nil, field_name: nil, label: nil, placeholder: 'Enter text', width: 'w-56', left_icon: nil,
-                  right_icon: nil, type: 'text_field', options: [], value: 'nil')
+  def input_field(form: nil, field_name: nil, text_field_name: nil, radio_field_name: nil, label: nil,
```

Comment:
why do we need field_name, text_field_name and radio_field_name ? input_field shouldn't abstract all sort of inputs ? For each input components we need to have separate components. Merging everything to a single component makes it very complicated.